spec/spec.md

```diff
@@ -396,274 +371,339 @@ values, and an explanation why a certain item or set of data is being requested:
+
+### Embed Targets
+
+The following section details where the _Presentation Submission_ is to be
+embedded within a target data structure, as well as how to formulate the
+[JSONPath](https://goessner.net/articles/JsonPath/) expressions to select the
+[[ref:Claims]] within the target data structure.
+
+#### Embed Locations
+
+The following are the locations at which the `presentation_submission` object
+****MUST**** be embedded for known target formats. For any location besides
+the top level of the embed target, the location is described in JSONPath syntax.
+
+Target     | Location
+---------- | --------
+OpenID     | top-level
+DIDComms   | `$.presentations~attach.data.json`
+VP         | top-level
+CHAPI      | `$.data`
+
+## Claim Format Designations
+
+Within the _Presentation Exchange_ specification, there are numerous sections
+where [[ref:Verifiers]] and [[ref:Holders]] convey what [[ref:Claim]] variants
+they support and are submitting. The following are the normalized references
+used within the specification:
+
+- `jwt` - the format is a JSON Web Token (JWTs) [[spec:rfc7519]]
+  that will be submitted in the form of a JWT encoded string. Expression of
+  supported algorithms in relation to this format ****MUST**** be conveyed using
+  an `alg` property paired with values that are identifiers from the JSON Web
+  Algorithms registry [[spec:RFC7518]].
+- `jwt_vc`, `jwt_vp` - these formats are JSON Web Tokens (JWTs) [[spec:rfc7519]]
+  that will be submitted in the form of a JWT encoded string, and the body of
+  the decoded JWT string is defined in the JSON Web Token (JWT) [[spec:rfc7519]]
+  section of the
+  [W3C Verifiable Credentials specification](https://www.w3.org/TR/vc-data-model/#json-web-token).
+  Expression of supported algorithms in relation to these formats ****MUST****
+  be conveyed using an `alg` property paired with values that are identifiers
+  from the JSON Web Algorithms registry [[spec:RFC7518]].
+- `ldp` - the format is a Linked Data Proof [[spec:Linked Data Proofs]] that will 
+  be submitted as an object.
+  Expression of supported algorithms in relation to these formats ****MUST****
+  be conveyed using a `proof_type` property with values that are identifiers
+  from the
+  [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/).
+- `ldp_vc`, `ldp_vp` - Verifiable Credential Linked Data Proof and Verifiable Presentation Linked
+  Data Proof formats. These are descriptions of formats normatively defined in the W3C Verifiable
+  Credentials specification [[spec:VC-DATA MODEL]], and will be submitted in the form of a JSON
+  object. 
+  Expression of supported algorithms in relation to these formats ****MUST****
+  be conveyed using a `proof_type` property paired with values that are
+  identifiers from the
+  [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/).
+
+
+## Submission Requirement Feature
```

Comment:
would it work to nest all of the features into their own section?

Reply:
TODO @kimdhamilton

Reply:
TODO @kimdhamilton